src/app/subnets/views/SubnetsList/_index.scss

```diff
@@ -44,6 +44,10 @@
     }
   }
 
+  .subnet-group__select {
+    width: 9.375rem;
```

Comment:
Not sure how I feel about this, but since we'll be overhauling the page headers to be consistent with the machine list design, I'm happy to leave this here as long as you leave a "TODO" comment to remove this when the subnets page gets the treatment.